src/patternfly/components/Wizard/wizard.scss

```diff
@@ -27,122 +28,128 @@
   --#{$wizard}__nav-link--before--Height: 1.5rem;
-  --#{$wizard}__nav-link--before--Top: 0;
-  --#{$wizard}__nav-link--before--BackgroundColor: var(--#{$pf-global}--BackgroundColor--200);
-  --#{$wizard}__nav-link--before--BorderRadius: var(--#{$pf-global}--BorderRadius--lg);
-  --#{$wizard}__nav-link--before--Color: var(--#{$pf-global}--Color--100);
-  --#{$wizard}__nav-link--before--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$wizard}__nav-link--before--TranslateX: calc(-100% - var(--#{$pf-global}--spacer--sm));
-  --#{$wizard}__nav-link--m-current--before--BackgroundColor: var(--#{$pf-global}--active-color--100);
-  --#{$wizard}__nav-link--m-current--before--Color: var(--#{$pf-global}--Color--light-100);
+  --#{$wizard}__nav-link--before--Top: var(--#{$pf-global}--spacer--form-element); // TODO form element spacer
+  --#{$wizard}__nav-link--before--BackgroundColor: var(--pf-t--global--background--color--secondary--default);
+  --#{$wizard}__nav-link--before--BorderRadius: var(--pf-t--global--border--radius--large);
+  --#{$wizard}__nav-link--before--Color: var(--pf-t--global--text--color--regular);
+  --#{$wizard}__nav-link--before--FontSize: var(--pf-t--global--font--size--body--default);
+  --#{$wizard}__nav-link--before--TranslateX: calc(-100% - var(--pf-t--global--spacer--xs));
+  --#{$wizard}__nav-link--m-current--before--BackgroundColor: var(--pf-t--global--color--brand--default);
+  --#{$wizard}__nav-link--m-current--before--Color: var(--pf-t--global--text--color--on-brand--default);
   --#{$wizard}__nav-link--m-disabled--before--BackgroundColor: transparent;
-  --#{$wizard}__nav-link--m-disabled--before--Color: var(--#{$pf-global}--disabled-color--100);
+  --#{$wizard}__nav-link--m-disabled--before--Color: var(--pf-t--global--text--color--disabled);
 
   // Toggle
-  --#{$wizard}__toggle--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
-  --#{$wizard}__toggle--ZIndex: var(--#{$pf-global}--ZIndex--xs);
-  --#{$wizard}__toggle--BoxShadow: var(--#{$pf-global}--BoxShadow--md-bottom);
-  --#{$wizard}__toggle--PaddingTop: var(--#{$pf-global}--spacer--lg);
-  --#{$wizard}__toggle--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$wizard}__toggle--PaddingBottom: var(--#{$pf-global}--spacer--lg);
-  --#{$wizard}__toggle--PaddingLeft: calc(var(--#{$pf-global}--spacer--md) + var(--#{$wizard}__nav-link--before--Width) + var(--#{$pf-global}--spacer--sm));
-  --#{$wizard}__toggle--m-expanded--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$wizard}__toggle--m-expanded--BorderBottomColor: var(--#{$pf-global}--BorderColor--100);
-  --#{$wizard}--m-in-page__toggle--xl--PaddingLeft: calc(var(--#{$pf-global}--spacer--xl) + var(--#{$wizard}__nav-link--before--Width) + var(--#{$pf-global}--spacer--sm));
+  --#{$wizard}__toggle--BackgroundColor: var(--pf-t--global--background--color--primary--default);
+  --#{$wizard}__toggle--ZIndex: var(--pf-t--global--Zindex--xs);
```

Comment:
Toggle has a box shadow that isn't showing because it's behind the wizard nav - increasing the z-index on the toggle gets it to show. It's worth noting that v5 has this same z-index setup and the box shadow on the toggle does show, so I'm wondering if something else has changed the stacking order of toggle + nav on mobile?

Reply:
Confirmed this is pre-existing - it happens when the drawer is in the main section. It's probably best fixed by assigning a z-index to the main section that's lower than the toggle, which should keep the toggle on top of all children in the main section, regardless their z-index. However, the main section has no z-index on purpose so its children (like menus) that have z-indexes can pop out on top of the footer. I'm not sure how to get around that.

Another option could be to use a border instead of a shadow on the toggle, similar to the update I think we made for the wizard footer. There is already a border in place of the shadow on the toggle when the nav is expanded. wdyt @lboehling? It would look like this:

This is the current shadow:
<img width="957" alt="Screenshot 2024-02-13 at 9 48 45 PM" src="https://github.com/patternfly/patternfly/assets/35148959/9916d3f6-7d2d-47db-aca4-cbd02dcbd541">

With a border instead:
<img width="967" alt="Screenshot 2024-02-13 at 9 51 07 PM" src="https://github.com/patternfly/patternfly/assets/35148959/d191c92a-f956-488c-81e5-f61ea94a6338">

And the pre-existing border when the nav is expanded:
<img width="957" alt="Screenshot 2024-02-13 at 9 49 59 PM" src="https://github.com/patternfly/patternfly/assets/35148959/65f27de5-91bb-47ca-b0f9-0159c5a000fb">